association_backend/association/api/query/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .query import Query
+
+__all__ = ["Query"]
```

Comment:
remove this and just use query.py in the `api` directory

Reply:
ok done